### PR TITLE
Feat: Test for repay more than principal with external price

### PR DESCRIPTION
**Pull request by mustermeiszer**
# Description
Testing whether one can repay more than principal on external loan, if settlement price is higher essentially.

## Changes and Descriptions
* Just adds an additional test

# Checklist:

- [x] I have added Rust doc comments to structs, enums, traits and functions
- [x] I have made corresponding changes to the documentation
- [x] I have performed a self-review of my code
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by lemunozm** [comment]
I think the test you wanted to see is this: https://github.com/centrifuge/centrifuge-chain/blob/main/pallets/loans/src/tests/repay_loan.rs#L552

Could it be?

**Comment by mustermeiszer** [comment]
Nah, this line https://github.com/centrifuge/centrifuge-chain/blob/430fbe9df5ccd51936535bce3362c20818034fc1/pallets/loans/src/tests/repay_loan.rs#L553 should provide a lower price than the one provided by the repayment input.

**Review by lemunozm — APPROVED** [review]
Yeah, then you're right!

Just a comment. But I'm fine adding the test

File: pallets/loans/src/tests/repay_loan.rs [hunk 1]
```diff
@@ -812,3 +812,78 @@ fn with_unregister_price_id_and_oracle_not_required() {
 		);
 	});
 }
+
+#[test]
+fn with_more_than_required_external() {
+	new_test_ext().execute_with(|| {
+		let mut pricing = util::base_external_pricing();
+		pricing.max_price_variation = Rate::from_inner(99_990_000_000_000_000_000_000_000_000);
```
[the remark this answers is not in the document]
Comment (mustermeiszer) [reply pallets/loans/src/tests/repay_loan.rs:820]: Yeah. Haha. I noticed that too. Will adapt!

**Comment by mustermeiszer** [comment]
Close in favor of https://github.com/centrifuge/centrifuge-chain/pull/1812/commits/fcddabf7f6eabde24ac2d0b89b7863ff4d06e06d

**Pull request closed without merging**